### DISCUSSION TITLE
Creating mania difficulty set only 5K default

### DISCUSSION THREAD

**Discussion by BKVad1m** [ppy/osu#20525] [Ideas] [answered]
![osu_2022-09-29_10-23-24](https://user-images.githubusercontent.com/76803825/192931478-0554034b-32bc-4eff-99df-eff76c03a8a3.png)
I requesting select keys if you need before creating difficulty

**Comment by ghost** [comment]
(You can change the keycount via the "circle size" slider in setup, but I do agree that it should be renamed to reflect its actual use)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/12904
